### DISCUSSION TITLE
Is it possible to disable RTC text overlay on RTC feed?

### DISCUSSION THREAD

**Discussion by simondsmason** [blakeblackshear/frigate#8742] [Ask A Question]
Is is possible to disable this tag in the top right hand corner?  And if so, how?  Thanks.

<img width="99" alt="image" src="https://github.com/blakeblackshear/frigate/assets/6063262/d7637202-1a5a-41f3-a379-38d6ea011410">

**Comment by NickM-27** [comment]
where are you viewing this stream?

**Comment by simondsmason** [comment]
I am using the following URL:

http://[myserver]:5000/live/webrtc/stream.html?src=driveway&mode=webrtc

**Reply by NickM-27** [reply]
this is added by go2rtc, there is no way to control it

**Reply by simondsmason** [reply]
Thanks, I assumed that.  I will try posting over there to see if there is a switch to disable it.  This particular feed from frigate is working well for me but I have no need for RTC on top of all of my feeds.

**Reply by simondsmason** [reply]
One additional question - is there a WebRTC command line switch to mute the stream by default?  I can't seem to find anything on switches as I search online.  Thanks.

**Comment by simondsmason** [comment]
I did look into this further.  Apparently you can suppress the RTC label by adding the style: '.mode {display: none}' to the ui settings for go2rtc.  Since go2rtc was installed by Frigate I am not sure where these settings might be or if they are accessible?
